base/some.jl

```diff
@@ -95,3 +97,23 @@ something() = throw(ArgumentError("No value arguments present"))
 something(x::Nothing, y...) = something(y...)
 something(x::Some, y...) = x.value
 something(x::Any, y...) = x
+
+#Methods for broadcast
+Base.getindex(s::Some) = s.value
```

Comment:
```suggestion
getindex(s::Some) = s.value
```

Don't need the `Base.`s here and below

Reply:
Whoops, that was some overzealous copy-pasting from my piracy experiment.

Reply:
I frequently use

```Julia
@eval Base begin
# experiments
end
```

Reply:
Good idea, thanks!